### PR TITLE
Conversation list mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this gets the conversation list dumb rendering done

screen shot 2017-02-10 at 3 14 45 pm

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 6f75f6550acc...dd061681f7ba introduce visual changes on linux. <details><summary>:mag_right: 9 changed</summary>

 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-You_Rekey_Selected-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-You_Rekey_Selected-dd061681f7ba.png)  
   Chat_Conversations_List-You_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-You_Rekey-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-You_Rekey-dd061681f7ba.png)  
   Chat_Conversations_List-You_Rekey comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Selected_Normal-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Selected_Normal-dd061681f7ba.png)  
   Chat_Conversations_List-Selected_Normal comparison
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Selected_Muted-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Selected_Muted-dd061681f7ba.png)  
   Chat_Conversations_List-Selected_Muted comparison
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Part_Rekey_Selected-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Part_Rekey_Selected-dd061681f7ba.png)  
   Chat_Conversations_List-Part_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Part_Rekey-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Part_Rekey-dd061681f7ba.png)  
   Chat_Conversations_List-Part_Rekey comparison
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Normal-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Normal-dd061681f7ba.png)  
   Chat_Conversations_List-Normal comparison
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Top-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Top-dd061681f7ba.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Bottom-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Bottom-dd061681f7ba.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486760406163-6f75f6550acc-dd061681f7ba-linux/Chat_Conversations_List-Long_Bottom.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 6f75f6550acc...a9eb9bc1a79d introduce visual changes on linux. <details><summary>:mag_right: 9 changed</summary>

 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-You_Rekey_Selected-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-You_Rekey_Selected-a9eb9bc1a79d.png)  
   Chat_Conversations_List-You_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-You_Rekey-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-You_Rekey-a9eb9bc1a79d.png)  
   Chat_Conversations_List-You_Rekey comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Selected_Normal-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Selected_Normal-a9eb9bc1a79d.png)  
   Chat_Conversations_List-Selected_Normal comparison
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Selected_Muted-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Selected_Muted-a9eb9bc1a79d.png)  
   Chat_Conversations_List-Selected_Muted comparison
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Part_Rekey_Selected-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Part_Rekey_Selected-a9eb9bc1a79d.png)  
   Chat_Conversations_List-Part_Rekey_Selected comparison
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Part_Rekey-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Part_Rekey-a9eb9bc1a79d.png)  
   Chat_Conversations_List-Part_Rekey comparison
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Normal-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Normal-a9eb9bc1a79d.png)  
   Chat_Conversations_List-Normal comparison
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Top-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Top-a9eb9bc1a79d.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Bottom-6f75f6550acc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Bottom-a9eb9bc1a79d.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486777024552-6f75f6550acc-a9eb9bc1a79d-linux/Chat_Conversations_List-Long_Bottom.png)
</summary>

File: shared/common-adapters/markdown.js [hunk 1]
```diff
@@ -53,7 +52,13 @@ class EmojiIfExists extends PureComponent<void, EmojiProps, void> {
   render () {
     const emoji = (this.props.children && this.props.children.join('')) || ''
     const exists = emojiIndex.emojis.hasOwnProperty(emoji.split(':')[1])
-    return exists ? <Emoji {...this.props} /> : <Text type='Body' style={this.props.preview ? neutralPreviewStyle : neutralStyle}>{emoji}</Text>
+    return exists
+      ? <Emoji {...this.props} />
+      : <Text
+        type='Body'
```
Comment (chromakode) [inline shared/common-adapters/markdown.js]: Indent

File: shared/local-debug-live.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ import {envVarDebugJson, isTesting} from './local-debug'
 
 // Shared settings
 const dumbFilterJson = (envVarDebugJson() || {}).dumbFilter || ''
-const dumbFilterOverride = '' // Changing this will apply during a hot reload session
+const dumbFilterOverride = 'conversations' // Changing this will apply during a hot reload session
```
Comment (chromakode) [inline shared/local-debug-live.js]: :scissors:

File: shared/local-debug-live.js [hunk 3]
```diff
@@ -8,10 +8,10 @@ import {envVarDebugJson, isTesting} from './local-debug'
...
 
 export const dumbFilter = dumbFilterOverride || dumbFilterJson
 
 // Mobile only settings
-export const dumbIndex = 0
+export const dumbIndex = 9
```
Comment (chromakode) [inline shared/local-debug-live.js]: :scissors:

**Pull request merged**